neurom/features/tests/test_neuronfunc.py

```diff
@@ -176,7 +176,7 @@ def test_sholl_crossings_simple():
     nt.eq_([2, 4, 5],
            list(_nf.sholl_crossings(SIMPLE, center, radii=radii)))
     
-    nt.eq_([2, 2, 2],
+    nt.eq_([1, 1, 2],
```

Comment:
@arnaudon please verify this change

Reply:
I think the ordering of sections changed, no?

for `SIMPLE.sections[:2]` in v1 I have:
```
(Section(id=0, type=NeuriteType.soma, n_points=2) <parent: None, nchildren: 0>,
 Section(id=1, type=NeuriteType.basal_dendrite, n_points=2) <parent: None, nchildren: 2>)
```
and for v2:
```[Section(id=0, type=3, n_points=2)<parent: Section(id=None), nchildren: 2>,
 Section(id=1, type=3, n_points=2)<parent: Section(id=0), nchildren: 0>]
```

Reply:
so we can close that